pilot/test/mock/config.go

```diff
@@ -596,11 +592,13 @@ func CheckCacheFreshness(cache model.ConfigStoreCache, namespace string, t *test
 		t.Error(err)
 	}
 
-	util.Eventually("successfully set done", func() bool {
-		doneMu.Lock()
-		defer doneMu.Unlock()
-		return done
-	}, t)
+	timeout := time.After(10 * time.Second)
```

Comment:
What was the total timeout before with eventually?

Reply:
Total waiting time will be about 1min. But for this test, I don't expect it to be that long. 

I put 60s in controller test. https://github.com/istio/istio/pull/4148/files